.github/workflows/prove_blocks.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         env:
           PATHFINDER_RPC_URL: ${{ secrets.PATHFINDER_RPC_URL }}
         run: |
-          cargo test --release --package prove_block --test prove_block -- test_prove_selected_blocks --show-output --ignored
+          cargo test --release --package prove_block --test prove_block -- --show-output --ignored
```

Comment:
Actually, I don't like how these ignored tests are used in the first place.

Reply:
Then just make the test conditional on the presence of `PATHFINDER_RPC_URL`? Seems you can do that with [feature flags](https://stackoverflow.com/questions/73760692/ignore-a-test-based-on-environment-variable).

Reply:
No, for this exact line we're commenting on! It can just be 
```
cargo test --release --package prove_block --test prove_block -- --show-output
```
and it will run the tests in release mode for just the `prove_block` tests.

Reply:
It's not a change for this PR anyway.